### PR TITLE
[Docs][HL Indy] Updated Troubleshooting Guide with Indy Troubleshooting

### PR DESCRIPTION
**Pull request by sauveergoel**
Signed-off-by: sauveergoel <sauveer.goel@accenture.com>

**Changelog**
- Added Indy Troubleshooting Guide section 

**Reviewed by**
@sownak @suvajit-sarkar @Pri-vats  

**Linked issue**
#885

### CODE REVIEW

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]
- [ ] The table reference alphabet should be changed from 'I' to something else

- [ ] The last coloumn should mention the precise steps (refer Fabric table)

File: docs/source/operations/fabric_verify.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-# How to debug a Fabric deployment
+# How to debug a BAF deployment
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:1]: File name needs to be changed as this is not only "fabric_verify" now

File: docs/source/operations/fabric_verify.md [hunk 2]
```diff
@@ -78,66 +78,162 @@ The flow chart shows the Fabric Deployment process. To verify the steps of deplo
...
+
+The flow chart shows the Indy Deployment process. To verify the steps of deployment, follow the Verification Table 'I', to troubleshoot the general errors.
+
+![](./../_static/indy_flowchart.png)
 
-Use this sample template.
-```yaml
+----
+### Indy Troubleshooting
+### Table 'I'
```
Comment (sownak) [inline docs/source/operations/fabric_verify.md:157]: Flowchart steps are wrong. A flow chart Process box should give an instruction like "Do x", "Check y", not just "Domain Genesis". Please check how it is worked in Fabric (difference between pods and jobs).

**Review by sownak — CHANGES_REQUESTED** [review]

File: docs/source/operations/fabric_verify.md (hunk 2 above)
Comment (sauveergoel) [reply docs/source/operations/fabric_verify.md]: done

File: docs/source/operations/fabric_verify.md (hunk 1 above)
Comment (sauveergoel) [reply docs/source/operations/fabric_verify.md:1]: done

**Comment by sauveergoel** [comment]
> * [ ]  The table reference alphabet should be changed from 'I' to something else
> * [ ]  The last coloumn should mention the precise steps (refer Fabric table)

DONE

**Review by jagpreetsinghsasan — COMMENTED** [review]
https://sauveer-baf.readthedocs.io/en/feature-helm3/operations/fabric_verify.html
This URL isnt accessible

**Comment by sauveergoel** [comment]
> https://sauveer-baf.readthedocs.io/en/feature-helm3/operations/fabric_verify.html
> This URL isnt accessible

Updated URL: https://sauveer-baf.readthedocs.io/en/feature-helm3/operations/baf_verify.html

**Review by sownak — APPROVED** [review]
LGTM

**Pull request merged**